integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -46,6 +46,11 @@ <h1 class="heading">
             </h1>
             {% if not page_form.instance.id or not page_form.instance.archived %}
                 <div class="flex flex-wrap grow justify-end gap-2 items-center">
+                    {% has_perm 'cms.publish_page_object' request.user page as can_publish_page %}
+                    {% if can_publish_page %}
+                        <a href="{% url 'pages' region_slug=request.region.slug %}"
+                           class="btn btn-ghost">{% translate "Cancel" %}</a>
+                    {% endif %}
```

Comment:
We're almost there :) Could you please push it beneath the `generic_auto_save_note`, because now it looks like this? :sweat_smile: :smiley: 
![grafik](https://github.com/user-attachments/assets/93739f3e-706b-4293-917f-518bd3361a8a)

Reply:
:see_no_evil:

Reply:
Looks good now :)